.github/workflows/ci-e2e.yml

```
@@ -91,7 +91,7 @@ jobs:
 
     cypress:
         name: Cypress E2E tests (${{ strategy.job-index }})
-        runs-on: ubuntu-24.04
+        runs-on: depot-ubuntu-24.04-64
```

woah there 🤣 
these runners are 32x more expensive 🤣 
can we start with a cheaper attempt
```suggestion
        runs-on: depot-latest-4
```

---

that has 16GB RAM
the `-64` has 256GB RAM 🙈

---

haha just trying to get across the line, and then scale back